drivers/usb/device/usb_dc_mcux_ehci.c

```diff
@@ -0,0 +1,490 @@
+/* usb_dc_ehci.c - EHCI OTG usb device driver */
+
+/*
+ * Copyright (c) 2018 NXP Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <soc.h>
+#include <string.h>
+#include <usb/usb_dc.h>
+#include <soc.h>
+#include <device.h>
+#include "usb_dc_mcux_nxp.h"
+#include "usb_device_ehci.h"
+#include <logging/sys_log.h>
```

Comment:
Need to switch to new logging system. This will become:
```
#include <logging/log.h>
LOG_MODULE_REGISTER(usb_dc_mcux_ehci, CONFIG_USB_DRIVER_LOG_LEVEL);
```

Reply:
@MaureenHelm thanks, I have updated it.